src/api-calls.js

```diff
@@ -0,0 +1,11 @@
+export default (path) => {
```

Comment:
I like how we imported this so that we don't have the same code being written time and time again